optimum/onnxruntime/modeling_diffusion.py

```diff
@@ -562,6 +565,15 @@ class ORTStableDiffusionInpaintPipeline(ORTStableDiffusionPipelineBase, StableDi
     __call__ = StableDiffusionInpaintPipelineMixin.__call__
 
 
+@add_end_docstrings(ONNX_MODEL_END_DOCSTRING)
+class ORTLatentConsistencyModelPipeline(ORTStableDiffusionPipelineBase, LatentConsistencyPipelineMixin):
```

Comment:
Could you add this one in the documentation?

Reply:
added it in https://github.com/huggingface/optimum/pull/1469/commits/824fc5702532eed226c139fdeba897be9cb7be16